packages/sparrowql-graphql/src/graph.ts

```diff
@@ -67,13 +67,13 @@ export function astToOptions(
           local: null,
           // TODO: Typings do not match for NamedType
           // eslint-disable-next-line @typescript-eslint/explicit-module-boundary-types, @typescript-eslint/no-unsafe-member-access
-          to: type2collection[(node.type as any).type.name.value as string],
+          to: type2collection[node.type.type.name.value],
         };
 
         for (const { name, value } of directive.arguments ?? []) {
           // Typings do not match for VariableNode
           if (name.value === 'as') {
-            relation.to = (value as ValueNodeWithValue).value.toString();
+            relation.to = value.value.toString();
```

Comment:
Similar situation here. `value` is of type `ValueNode` which is a union of a lot of types, but only the following types have the `.value` property. Same question as above - how should the logic look like with a typeguard?

```ts
type ValueNodeWithValue = Exclude<
  ValueNode,
  VariableNode | NullValueNode | ListValueNode | ObjectValueNode
>;
```

Reply:
`as` is an argument that we expect to be a `string` - we can assert it, i.e., throw an error if it's not a `StringValueNode`.